test/shopify_import_massager_test.exs

```diff
@@ -5,16 +5,33 @@ defmodule ShopifyImportMassagerTest do
 
   describe "gets substitutions and converts files" do
     test "converts test_orders_1|2|3" do
-      input_folder = "test/source_files"
-      output_folder = "test/output_files"
+      parameters = %{
```

Comment:
Aside from a change in how the conversion gets called, the tests are essentially unchanged. This is a good spot to be in when refactoring code as we want to have the confidence that altering the way things get done still works the way we expect.